java-frontend/src/main/java/org/sonar/java/AbstractJavaClasspath.java

```diff
@@ -51,7 +51,7 @@
 public abstract class AbstractJavaClasspath implements BatchExtension {
 
   private static final char SEPARATOR = ',';
-  private static final Logger LOG = LoggerFactory.getLogger(AbstractJavaClasspath.class);
+  private static final Logger LOG = Loggers.get(AbstractJavaClasspath.class);
   protected final Project project;
```

Comment:
![MINOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-minor.png) Remove this use of "Project"; it is deprecated. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](https://dory.sonarsource.com/sonarqube/coding_rules#rule_key=squid%3ACallToDeprecatedMethod)